### PR TITLE
Enable NVPTX LLVM target for v5

### PR DESCRIPTION
**Pull request by chrisburr**
Required for the next version of ROOT (and possibly cling?). See https://github.com/conda-forge/root-feedstock/pull/64.

Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by chrisburr** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
Maybe we should just rebuild it with all targets like was done in PR ( https://github.com/conda-forge/llvmdev-feedstock/pull/59 )?

**Comment by chrisburr** [comment]
@conda-forge-admin, please rerender

**Comment by chrisburr** [comment]
> Maybe we should just rebuild it with all targets like was done in PR ( #59 )?

Seems reasonable to me. Done!

@conda-forge/llvmdev This is ready to merge. The CI has already passed in ff99f0f16aae29c66d9d81e16404c42e00ae20be and the subsequent commits were only to disable CircleCI as is was much slower to build than Azure.

**Comment by chrisburr** [comment]
Ping @conda-forge/llvmdev

**Comment by chrisburr** [comment]
@conda-forge/llvmdev Sorry to be pushy but is there any chance this can be merged?

File: recipe/build.sh [hunk 1]
```diff
@@ -3,7 +3,6 @@ cd build
 
 cmake -DCMAKE_INSTALL_PREFIX="${PREFIX}" \
       -DCMAKE_BUILD_TYPE=Release \
-      -DLLVM_TARGETS_TO_BUILD=host \
```
Comment (isuruf) [inline recipe/build.sh]: Can you do the same in windows?

Comment (chrisburr) [reply recipe/build.sh]: It's already like this: https://github.com/conda-forge/llvmdev-feedstock/blob/2f5ed79c873ee4395dbc3d715868a542b8877022/recipe/bld.bat

**Comment by chrisburr** [comment]
Thanks @isuruf and @jakirkham! 🍰

**Comment by jakirkham** [comment]
Thanks for the addition, @chrisburr. Does this mean ROOT will have GPU support now? 😉

**Comment by chrisburr** [comment]
Of course! 😆 I don't really use ROOT but AFAIK it's just TMVA (machine learning) for now but it's also coming to RooFit (likelihood fitting).

I'll have to make a GPU variant at some point...

**Comment by jakirkham** [comment]
Cool! Let me know if you have any questions. 😉

**Pull request merged**